### PR TITLE
docs: Updating annotations and tutorial with new annotations

### PR DESCRIPTION
**Pull request by akshaysngupta**
This PR updates
1) `annotations.md` to have table of all the supported annotations along with newly added annotations.
2) `tutorial.md` to have an example for annotation. I selected cookie based affinity to just have 1 example.
Then we can say that here is a link to all the supported annotations.

### CODE REVIEW

File: docs/annotations.md [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
 Kubernetes Ingress specification allows for annotations. We use annotations to expose Application Gateway specific features that can't be exposed using the ingress specification. It is important to note that annotations defined on an ingress resource are applied to all HTTP setting, backend pools and listeners defined within a given ingress resource.
 
-## Backend path prefix
+### List of supported annotations
+| Annotation Key | Value Type | Default Value |
+| -- | -- | -- |
+| [appgw.ingress.kubernetes.io/backend-path-prefix](#backend-path-prefix) | `string` | `nil` |
+| [appgw.ingress.kubernetes.io/ssl-redirect](#ssl-redirect) | `bool` | `false` |  |
+| [appgw.ingress.kubernetes.io/connection-draining](#connection-draining) | `bool` | `false` |
+| [appgw.ingress.kubernetes.io/connection-draining-timeout](#connection-draining) | `int32` (seconds) | `30` |
```
Comment (asridharan) [inline docs/annotations.md]: Should we mention that `default` is 30? Just to be explicit.

File: docs/annotations.md [hunk 2]
```diff
@@ -73,3 +77,90 @@ spec:
           serviceName: websocket-repeater
           servicePort: 80
 ```
+
+# Connection Draining
+`connection-draining`: This annotations allows to specify whether to enable connecitn draining and the timeout after which
```
Comment (asridharan) [inline docs/annotations.md]: s/annotations/annotation

A few typos:
`connection-draining`: This annotation allows a user to specify connection draining and timeout ....

File: docs/annotations.md [hunk 3]
```diff
@@ -73,3 +77,90 @@ spec:
...
+`connection-draining-timeout`: This annotations allows to specify a timeout after which Application Gateway will terminate the requests to the draining backend endpoint.
```
Comment (asridharan) [inline docs/annotations.md]: s/annotations/annotation

This annotation allows a user to specify a timewout after which the Application Gateway will terminate the request to the draining backend.

File: docs/tutorial.md [hunk 4]
```diff
@@ -237,9 +239,30 @@ For any property that can not be inferred by the readiness/liveness probe, Defau
 | `Interval` | 30 |
 | `UnhealthyThreshold` | 3 |
 
+## Enable Cookie based Affinity
+As outlined in the [documentations](https://docs.microsoft.com/en-us/azure/application-gateway/application-gateway-components#http-settings), Application Gateway supports cookie based affinity enabling which it can direct subsequent traffic from a user session to the same server for processing.
```
Comment (asridharan) [inline docs/tutorial.md]: s/documentations/Azure Application Gateway Documentation/

File: docs/tutorial.md [hunk 5]
```diff
@@ -237,9 +239,30 @@ For any property that can not be inferred by the readiness/liveness probe, Defau
...
+      paths:
+      - backend:
+          serviceName: frontend
+          servicePort: 80
+```
+
 ## Expose a WebSocket server
 
-As outlined in the Application Gateway v2 documentation - it [provides native support for the WebSocket and HTTP/2 protocols](https://docs.microsoft.com/en-us/azure/application-gateway/overview#websocket-and-http2-traffic). For both App Gateway and the Kubernetes Ingress - there is no user-configurable setting to selectively enable or disable WebSocket support.
+As outlined in the Application Gateway v2 documentation - it [provides native support for the WebSocket and HTTP/2 protocols](https://docs.microsoft.com/en-us/azure/application-gateway/overview#websocket-and-http2-traffic). For both Application Gateway and the Kubernetes Ingress - there is no user-configurable setting to selectively enable or disable WebSocket support.
```
Comment (asridharan) [inline docs/tutorial.md]: s/For both/Please note, that for both

File: docs/annotations.md (hunk 1 above)
Comment (akshaysngupta) [reply docs/annotations.md]: The column heading is `Default Value`. Should we default in the cell as well ?

Comment (asridharan) [reply docs/annotations.md]: Missed that . We are good !! 👍

File: docs/annotations.md [hunk 6]
```diff
@@ -73,3 +77,90 @@ spec:
           serviceName: websocket-repeater
           servicePort: 80
 ```
+
+# Connection Draining
+`connection-draining`: This annotation allows to specify whether to enable connecitn draining.
```
Comment (asridharan) [inline docs/annotations.md]: s/connecitn/connection

Comment (akshaysngupta) [reply docs/annotations.md]: thanks!

**Pull request merged**